src/domain/templates/components/Forms/common/mappings.ts

```
@@ -134,220 +151,285 @@ export const toCreateTemplateMutationVariables = (
+        },
+      };
+      callout.contributionDefaults = handleContributionDefaults(calloutTemplateData.callout.contributionDefaults);
 
-        if (calloutDraft.callout?.framing) {
-          handleTags(calloutDraft.callout.framing);
-          handleReferences(calloutDraft.callout.framing);
-          delete calloutDraft.callout.framing['id'];
+      switch (calloutTemplateData.callout?.type) {
+        case CalloutType.Post:
+        case CalloutType.PostCollection: {
+          delete callout.contributionDefaults?.whiteboardContent;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)